Source/WebKit/Configurations/WebKit.xcconfig

```diff
@@ -243,8 +243,13 @@ WK_WRITING_TOOLS_LDFLAGS_maccatalyst_MACOS_SINCE_1500 = -weak_framework WritingT
 WK_WRITING_TOOLS_LDFLAGS_macosx = $(WK_WRITING_TOOLS_LDFLAGS$(WK_MACOS_1500));
 WK_WRITING_TOOLS_LDFLAGS_MACOS_SINCE_1500 = -weak_framework WritingTools;
 
+WK_USERNOTIFICATIONS_LDFLAGS = $(WK_USERNOTIFICATIONS_LDFLAGS_$(WK_PLATFORM_NAME));
+WK_USERNOTIFICATIONS_LDFLAGS_ = ;
```

Comment:
I think you can remove this line. For platform names other than the three you list below, the build setting will be implicitly empty by default.